libraries/AP_RangeFinder/AP_RangeFinder_LightWareSerial.cpp

```diff
@@ -74,8 +77,8 @@ bool AP_RangeFinder_LightWareSerial::get_reading(float &reading_m)
             } else {
                 // received the low byte which should be second
                 if (high_byte_received) {
-                    const float dist = (high_byte & 0x7f) << 7 | (c & 0x7f);
-                    if (!is_negative(dist)) {
+                    const int16_t dist = (high_byte & 0x7f) << 7 | (c & 0x7f);
```

Comment:
This vexes me a little - this should be `uint16_t`.

However, I would wager if we were to change it here we'd get signed/unsigned warnings when we compare against the maximum distance parameter, which *is* unsigned.  Correct fix would probably be to clamp the parameter value.  Unrelated problem, and way too complicated to fix in this PR :-)